aspnetcore/mvc/controllers/areas.md

```diff
@@ -45,119 +45,88 @@ Let's take a look at an example to illustrate how Areas are created and used. Le
+For more information, see [Area routing](xref:mvc/controllers/routing#areas).
 
-* Generating links from an action within an area based controller to another action on a different controller
+### Link Generation with Areas
 
-  Let's say the current request's path is like `/Products/Home/Create`
+The following code from the [sample download](https://github.com/aspnet/Docs/tree/master/aspnetcore/mvc/controllers/areas/samples) shows link generation with the area specified:
 
-  HtmlHelper syntax: `@Html.ActionLink("Go to Manage Products Home Page", "Index", "Manage")`
+[!code-cshtml[](areas/samples/MVCareas/Views/Shared/_testLinksPartial.cshtml?name=snippet)]
 
-  TagHelper syntax: `<a asp-controller="Manage" asp-action="Index">Go to Manage Products Home Page</a>`
+The links generated with the preceding code are valid anywhere in the app.
 
-  Note that here the ambient value of an 'area' is used but the 'controller' value is specified explicitly above.
+The sample download includes a [partial view](xref:mvc/views/partial) that contains the preceding links and the same links without specifying the area. The partial view is referenced in the [layout file](), so every page in the app displays the generated links. The links generated without specifying the area are only valid when referenced from a page in the same area and controller.
 
-* Generating links from an action within an area based controller to another action on a different controller and a different area.
+When the area or controller is not specified, routing depends on the `ambient` values. The current route values of the current request are considered ambient values for link generation. In many cases for the sample app, using the ambient values generates incorrect links.
 
-  Let's say the current request's path is like `/Products/Home/Create`
+For more information, see [Routing to controller actions](xref:mvc/controllers/routing).
 
-  HtmlHelper syntax: `@Html.ActionLink("Go to Services Home Page", "Index", "Home", new { area = "Services" })`
+### Shared layout for Areas using the _ViewStart.cshtml file
 
-  TagHelper syntax: `<a asp-area="Services" asp-controller="Home" asp-action="Index">Go to Services Home Page</a>`
+To share a common layout for the entire app, move the *_ViewStart.cshtml* to the application root folder.
 
-  Note that here no ambient values are used.
+<!-- This section will be completed after https://github.com/aspnet/Docs/pull/10978 is merged.
+<a name="arp"></a>
 
-* Generating links from an action within an area based controller to another action on a different controller and **not** in an area.
+## Areas for Razor Pages
+-->
+<a name="rename"></a>
 
-  HtmlHelper syntax: `@Html.ActionLink("Go to Manage Products  Home Page", "Index", "Home", new { area = "" })`
+### Change default area name
```

Comment:
Is it really changing the area name or the folder where area views are stored?

IMO an area name is something like "Products" or "Forums" that is in `[Area("Forums")]`.